packages/pigeon/CHANGELOG.md

```diff
@@ -1,7 +1,3 @@
-## 4.2.15
```

Comment:
In the future please don't land straight reverts of package updates that have been published, as it creates issues for the next version bump. This should have been 4.2.16 instead.

I've retracted 4.2.15 so we don't have a version in the wild that we no longer have source for, but please do a new PR that restores this changelog entry, with a note saying it was retracted, and restores 4.2.15 as the version, so that the next PR will, correctly, change it to 4.2.16.

Reply:
I filed https://github.com/flutter/flutter/issues/117458 because this should be enforced by CI, rather than relying on everyone contributing to flutter/packages and flutter/plugins somehow knowing this.

Reply:
Got it, sorry about that.